tsconfig.json

```diff
@@ -14,15 +18,40 @@
     "isolatedModules": true,
     "jsx": "preserve",
     "incremental": true,
+    "baseUrl": ".",
+    "paths": {
+      "src/*": [
+        "src/*"
+      ],
+      "@widgets/*": [
+        "src/widgets/*"
+      ],
+      "@features/*": [
+        "src/features/*"
+      ],
+      "@shared/*": [
+        "src/shared/*"
+      ],
+      "@public/*": [
```

Comment:
check the public path.
https://blog.logrocket.com/using-path-aliases-cleaner-react-typescript-imports/